crates/ra_ide_db/Cargo.toml

```diff
@@ -26,6 +26,7 @@ ra_text_edit = { path = "../ra_text_edit" }
 ra_db = { path = "../ra_db" }
 ra_prof = { path = "../ra_prof" }
 test_utils = { path = "../test_utils" }
+ra_hir_def = { path = "../ra_hir_def" }
```

Comment:
this crates should not depend on anything bellow ra_hir

Reply:
(Note to self): Pending https://github.com/rust-analyzer/rust-analyzer/pull/4873#pullrequestreview-459254605